piezo_web_app/PiezoWebApp/src/services/application_builder/manifest_populator.py

```diff
@@ -0,0 +1,62 @@
+class ManifestPopulator:
+
+    def build_manifest(self, validated_parameters_dict):
+        manifest = self.default_spark_application_manifest()
```

Comment:
This (and other methods below) are marked as `@staticmethod` so you'll need to call, e.g. `ManifestPopulator.default_spark_application_manifest()` instead